config/fam_cis_config

```diff
@@ -0,0 +1,20 @@
+#CIS Config file
```

Comment:
Leave it as empty with comment - cis config parameters here. We can add the file  when we enable cis configs.